pkg/restore/restore.go

```diff
@@ -1468,3 +1471,140 @@ func isCompleted(obj *unstructured.Unstructured, groupResource schema.GroupResou
 	// Assume any other resource isn't complete and can be restored
 	return false, nil
 }
+
+type restoreableResource struct {
```

Comment:
Can you provide comments on what the differences are between a restoreableResource and a restoreableItem are here?

Reply:
added comment. can reword if this one sounds confusing.